README.md

```diff
@@ -11,10 +11,17 @@ Decode: python3 tfci.py decompress ../data/image_compression_benchmark/big_build
 # K-means clustering-based codec
 ## Run codec
 python3 kmeans.py [path of image to be compressed] compressed/[name of compressed image file] reconstructed/[name of decoded image file] [number of clusters]
+<!-- edit2 -->
+
 
 # Classical algorithms - JPEG/PNG
 ## Build codec
+<!-- edit 4 -->
```

Comment:
You have made 14 changes to the README.md file. Here is a review of each change:
1. `++ b/README.md` - This line indicates that you are adding new content to the README.md file in the repository.
2. `+++ b/README.md` - This line indicates that you are modifying existing content in the README.md file.
3. `Decode: python3 tfci.py decompress ../data/image_compression_benchmark/big_build` - This line suggests that you have added a new command for decoding images using the TFCI library in the repository.
4. `## K-means clustering-based codec` - This line indicates that you have added a new section to the README.md file for a K-means clustering-based image compression algorithm.
5. `##